optuna/_hypervolume/utils.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-
+from optuna.samplers._tpe.sampler import _compute_hypervolume
```

Comment:
We can get away with not having to depend on `optua.samplers._tpe` from this file. `_compute_hypervolume` is just a thin wrapper of `WFG.compute`, so I suggest removing this `_compute_hypervolume` altogether and import `WFG` directly here. What do you think?

Reply:
OK let me do that next

Reply:
done!

Reply:
wrapper removed... let me know if the code passes the new tests

Reply:
Sorry but I am having another circular import issue, with logging.  See below.
Pytest has its own logging module (in site-packages. _pytest/logging.py), which imports optuna's logging, which imports itself (?) ('import logging', again) then tries to get a variable from pytest logging module (from logging import CRITICAL) which has not yet been defined since it happens after the import (in _pytest.logging).  
When the debugger arrives in optuna/logging.py, there is no 'logging' in the vars, so importing CRITICAL is bound to fail.

`ImportError: cannot import name 'CRITICAL' from partially initialized module 'logging' (most likely due to a circular import) (/optuna/optuna/logging.py)`

The problem has to come from the 'import logging' in optuna/logging.py since it would be silly to import itself, but that's what the debugger does.  
Your code probably tries to import the global variables from 'somewhere' but it's not what's happening.  
I have tried to change pycharm setup to define either optuna/optuna as 'sources root', or 'tests', or 'optuna', and various combinations, nothing works.  
So I can't get to the circular problem you mention for now.  
Any suggestion?

Reply:
Hm, do you happen to have the entire stack trace?

Also, could you try to debug the error above in the master branch, without the changes in this PR? I understand that it's something caused by the environment and not really the changes in this PR branch.

Also, let me ask the other developers of Optuna if there are some PyCharm users.

Reply:
Hello, I just encountered this issue before and the cause is `logging.py` on the current directory.
I'm not sure whether it is related to your problem or not, please check that as well.

```sh
(venv) > ls
CODE_OF_CONDUCT.md LICENSE            asv.conf.json      docs               optuna             setup.cfg          tutorial
CONTRIBUTING.md    MANIFEST.in        benchmarks         examples           optuna.egg-info    setup.py           venv
Dockerfile         README.md          build              formats.sh         pyproject.toml     tests
(venv) > python3 -c 'import optuna'  # it works

(venv) > touch logging.py
(venv) > ls
CODE_OF_CONDUCT.md LICENSE            asv.conf.json      docs               logging.py         pyproject.toml     tests
CONTRIBUTING.md    MANIFEST.in        benchmarks         examples           optuna             setup.cfg          tutorial
Dockerfile         README.md          build              formats.sh         optuna.egg-info    setup.py           venv
(venv) > python3 -c 'import optuna'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/himkt/work/github.com/himkt/optuna/optuna/__init__.py", line 6, in <module>
    from optuna import logging
  File "/Users/himkt/work/github.com/himkt/optuna/optuna/logging.py", line 2, in <module>
    from logging import CRITICAL
ImportError: cannot import name 'CRITICAL' from 'logging' (/Users/himkt/work/github.com/himkt/optuna/logging.py)
(venv) >
```

Reply:
Yes, that's exactly the error I get with pytests... here's the trace
> "/Volumes/DATA/Dropbox/IMAC BACKUP/WORK/PROJECTS/OPTUNA/venv/bin/python" "/Volumes/DATA/Dropbox/COMMON/Applications/Utilities_Development/PyCharm 2021.3.app/Contents/plugins/python/helpers/pycharm/_jb_pytest_runner.py" --target hypervolume_tests/test_utils.py::test_solve_hssp
Testing started at 10:02 ...
Traceback (most recent call last):
  File "/Volumes/DATA/Dropbox/COMMON/Applications/Utilities_Development/PyCharm 2021.3.app/Contents/plugins/python/helpers/pycharm/_jb_pytest_runner.py", line 5, in <module>
    import pytest
  File "/Volumes/DATA/Dropbox/IMAC BACKUP/WORK/PROJECTS/OPTUNA/venv/lib/python3.9/site-packages/pytest/__init__.py", line 29, in <module>
    from _pytest.logging import LogCaptureFixture
  File "/Volumes/DATA/Dropbox/IMAC BACKUP/WORK/PROJECTS/OPTUNA/venv/lib/python3.9/site-packages/_pytest/logging.py", line 3, in <module>
    import logging
  File "/Volumes/DATA/Dropbox/IMAC BACKUP/WORK/PROJECTS/OPTUNA/optuna/optuna/logging.py", line 3, in <module>
    from logging import CRITICAL
ImportError: cannot import name 'CRITICAL' from partially initialized module 'logging' (most likely due to a circular import) (/Volumes/DATA/Dropbox/IMAC BACKUP/WORK/PROJECTS/OPTUNA/optuna/optuna/logging.py)

I have the same issue running test 'test_compute_2points_volume' from the master branch...

Reply:
I guess you need to "Unmark as Sources Root" for `optuna/optuna`. The sources root should be `optuna`, not `optuna/optuna`, I think.